docker-compose.yml

```
@@ -535,7 +535,7 @@ services:
       CELERY_BROKER_PASSWORD: password
       DJANGO_WATCHMAN_TIMEOUT: 30
       ANALYTICS_DASHBOARD_CFG: /edx/etc/registrar.yml
-    image: edxops/registrar-dev:${OPENEDX_RELEASE:-latest}
+    image: edxops/registrar-devstack-dev:${OPENEDX_RELEASE:-latest}
```

This change can be skipped if we decide to use `registrar-dev`.
```suggestion
    image: edxops/registrar-dev:${OPENEDX_RELEASE:-latest}
```